praw/models/reddit/mixins/__init__.py

```diff
@@ -134,10 +139,54 @@ def remove(self, spam=False):
            # remove a submission
            submission = reddit.submission(id='5or86n')
            submission.mod.remove()
+           # remove a submission with a removal reason
+           reason = reddit.subreddit.mod.removal_reasons["110ni21zo23ql"]
+           submission = reddit.submission(id="5or86n")
+           submission.mod.remove(reason_id=reason.id)
 
         """
         data = {"id": self.thing.fullname, "spam": bool(spam)}
         self.thing._reddit.post(API_PATH["remove"], data=data)
+        if any([reason_id, mod_note]):
+            self._add_removal_reason(mod_note, reason_id)
+
+    def _add_removal_reason(self, mod_note="", reason_id=None):
```

Comment:
Yes, adding an underscore prefix to the method is how to indicate a method is _private_. Now that you've done that, could you please sort this instance method name? Such methods usually go towards the top of the instance methods since `'_'` < `'A'`